helix-core/src/main/java/org/apache/helix/common/caches/ParticipantStateCache.java

```diff
@@ -57,10 +57,11 @@ public ParticipantStateCache(ControlContextProvider controlContextProvider) {
    * @param liveInstanceMap map of all liveInstances in cluster
    * @return
    */
-  public boolean refresh(HelixDataAccessor accessor, Map<String, LiveInstance> liveInstanceMap) {
+  public boolean refresh(HelixDataAccessor accessor, Map<String, LiveInstance> liveInstanceMap,
+      Set<String> restrictedKeys) {
```

Comment:
Following the previous comment, I think this restrictedKeys can be a field only for the customizedStateCache class. As long as we input it in the constructor, we will be able to read it in the customizedStateCache class method. For this parent class, it does no need to know it.